aries-site/src/pages/components/notifications.mdx

```diff
@@ -0,0 +1,129 @@
+import { StateVsStatusDiagram } from '../../examples';
+import { Box, Text } from 'grommet';
+
+### What are they for?
+
+Notifications build trust by providing users confidence and assurance that they are informed with timely, relevant status about their systems and tasks.
+
+## Taxonomy 
+
+To standardize our language on Notifications across platforms, be sure to familiarize yourself with this taxonomy:
+
+- Type - In the broadest terms, what the notification is about
+- State - The current stage within a process
+- Status - The condition of a process
+- Attention Level - The priority and importance of a notification
+- Actions - Interactactions within a notification
+- Persistence - How the notification is dismissed
+- Placement - Where the notification can be found
+- Insight - The content of the notification
+
+<Box direction="row-responsive" gap="medium" pad={{ top: 'medium' }}>
```

Comment:
Maybe we could implement this as a markdown table so it looks more like the tables on the Voice and Tone page. Might also cleanup the code for this page a little. Documentation: https://www.markdownguide.org/extended-syntax/#tables